cluster.go

```diff
@@ -863,6 +883,12 @@ func (c *Cluster) SyncAllLocal() ([]api.PinInfo, error) {
 	return syncedItems, err
 }
 
+// Sync triggers a LocalSyncCid() operation for a given Cid
```

Comment:
It looks like `LocalSyncCid()` was a potential new name for `SyncLocal()`, however, the name is not changed so this comment is a bit confusing.

Reply:
or a potential old name :P. fixed...